packages/sources/balance-price/src/endpoint/balance.ts

```diff
@@ -0,0 +1,54 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { config } from '../config'
+import overrides from '../config/overrides.json'
+import { balanceTransport } from '../transport/balance'
+
+export const inputParameters = new InputParameters(
+  {
+    addresses: {
+      array: true,
+      type: 'string',
+      required: true,
+      description: 'The addresses to check the balance of',
+    },
+    blockNumber: {
+      type: 'number',
+      required: false,
+      description: 'The block number to check the balance at',
+    },
+  },
+  [
+    {
+      addresses: [
+        '0x61E5E1ea8fF9Dc840e0A549c752FA7BDe9224e99',
```

Comment:
Can we change this part so that instead of array of strings, `addresses` accepts array of address objects, as per the requirement. 
```json
{
  "data": {
    "endpoint": "balance",
    "addresses": [
      {
        "address": "0x638884A3F4643Ab2a80afa8103Cd2B6Cc84f2a0c"
      },
      {
        "address": "0xC88F7666330b4b511358b7742dC2a3234710e7B1"
      }
    ]
  }
}
```